activerecord/lib/active_record/railties/databases.rake

```diff
@@ -449,16 +449,15 @@ db_namespace = namespace :db do
     task dump: :load_config do
       ActiveRecord::Tasks::DatabaseTasks.with_temporary_pool_for_each do |pool|
         db_config = pool.db_config
-        schema_format = ENV.fetch("SCHEMA_FORMAT", ActiveRecord.schema_format).to_sym
```

Comment:
See my previous comment about `SCHEMA_FORMAT`. I think we should only reference it in the database tasks if possible.